db/migrate/20150813115025_create_container_resource_quota.rb

```diff
@@ -0,0 +1,19 @@
+class CreateContainerResourceQuota < ActiveRecord::Migration
+  def change
+    create_table :container_quotas do |t|
+      t.string     :name
+      t.datetime   :creation_timestamp
+      t.string     :resource_version
+      t.string     :ems_ref
+      t.belongs_to :container_project, :type => :bigint
+      t.belongs_to :ems, :type => :bigint
+    end
+
+    create_table :container_quota_items do |t|
+      t.string     :resource
+      t.string     :quota
```

Comment:
@moolitayer move this to `quota_desiderd, quota_enforced, quota_observed` and use a `:bigint` running the value through the conversion function we have in the parser.